examples/js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "spectacle-example-js",
```

Comment:
Each of the examples is now a part of the workspace and have their own scripts/deps (most of the deps are just shared so no extra baggage, just have to declare them here). This allows us to run scripts in all of the examples at once, etc. - and allows us to link to the workspace spectacle code so we can update spectacle core code and have it reflected live-time in our examples (without any linking etc)